### ISSUE TITLE
Solutions Page : Lead space block & CTA section - on click, buttons are not operable

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#2910]
## Detailed description

Keyboard and screen reader issue.

Issue :- 
1. With tab/Shift tab focus is not going to the mentioned links
2. Screen reader issue :- In arrow key navigation and swipe navigation ( left and right swipe) this control is available twice for the screen reader user.

Expected :-
1. Focus should go to each and every interactive element and it should be operable thru keyboard.
2. screen reader should identify the control only once.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/27b8a92c-9dd9-46fd-88f9-245d38d1fae7)
